docs/gatsby-config.js

```diff
@@ -24,9 +24,13 @@ module.exports = {
           null: [
             'index',
             'getting-started',
-            '[Changelog](https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md)',
+            'integrations/middleware',
```

Comment:
I think this is a good move, but then as a follow-up PR we might want to rework this file to be more like "Which package should I use?". The docs are really missing a page that says something like,

`apollo-server` is the batteries-included version which is easy to set up but doesn't provide the ability to make lots of tweaks to HTTP serving and doesn't slot into an existing web server.  If you're not trying to put it into an existing web server then use it until you need to do something it doesn't support, and if you hit that point, switch to `apollo-server-express`. Here's the exact changes you have to make to turn an `apollo-server` server into an `apollo-server-express` server.  The other integrations are something you should use only if you are already using those web frameworks/serverless function providers and want to slot into it; otherwise you should stick with Express since it is by far the most used library.

You want to do this or me?